configurations.md

```diff
@@ -0,0 +1,1178 @@
+---
+---
+
+# User Configuration Files
+
+<div class="row">
+<div class="col-lg" markdown="1">
+
+## Developer Configurations
+
+Below are the configurations of a few of xmonad's maintainers.  Just
+keep in mind that these setups are very customized and perhaps a little
+bit hard to replicate (some may rely on features only available in
+personal forks or git), may or may not be documented, and most aren't
+very pretty either :)
+
+<div class="list-col-4" markdown="1">
+  - [byorgey](https://github.com/byorgey/dotfiles)
+  - [geekosaur](https://github.com/geekosaur/xmonad.hs/tree/pyanfar)
+  - [liskin](https://github.com/liskin/dotfiles/tree/home/.xmonad)
+  - [psibi](https://github.com/psibi/dotfiles/tree/master/xmonad)
+  - [slotThe](https://gitlab.com/slotThe/dotfiles/-/tree/master/xmonad/.config/xmonad)
+  - [TheMC47](https://github.com/TheMC47/dotfiles/tree/master/xmonad/.xmonad)
+</div>
+
+## Legacy Configurations
+
+These are currently part of the `xmonad-contrib` package but will be
+removed at some point in the future.  People importing them should copy
+the relevant functions into their own configurations.
+
+### XMonad.Config.Arossato
```

Comment:
Not sure if it's worth keeping these in separate files, but since they're not really going to change much I went with just copying them here